README.md

```diff
@@ -79,6 +79,24 @@ Provides a generic proxy to consume resources from other origins without causing
 
 URL-Format `/{http|https}/{host}/{path}`
 
+### remote location
+
+The environment variable REMOTE_LOCATION allows setting a remote location (similar to the behaviour of the JAVA proxy).
```

Comment:
I think it makes sense to remove the mentioning of the Java proxy here as it's not open source and people might get confused.